.github/workflows/coverage-report.yml

```diff
@@ -15,7 +15,7 @@ permissions:
 jobs:
   coverage-report:
     runs-on: ubuntu-latest
-    if: ${{ false && github.event.workflow_run.conclusion == 'success' }}
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

Comment:
Maybe we should do this in a separate PR where we can confirm that the problem with the coverage action is actually fixed

Reply:
Sounds like a good plan. If you pop one in I can drop the commit from this pr. Then go fix my fork if necessary!

Reply:
I'm honestly really not that motivated to have this functionality restored. I just don't want to start getting bogus workflow failure notification emails again.